app/authentication/authenticator.py

```diff
@@ -114,18 +114,18 @@ def store_session(metadata):
 
 
 def decrypt_token(encrypted_token):
-    logger.debug('decrypting token')
-    if not encrypted_token or encrypted_token is None:
+    if not encrypted_token:
         raise NoTokenException('Please provide a token')
 
+    logger.debug('decrypting token')
     decrypted_token = decrypt(token=encrypted_token,
                               key_store=current_app.eq['key_store'],
                               key_purpose=KEY_PURPOSE_AUTHENTICATION,
                               leeway=current_app.config['EQ_JWT_LEEWAY_IN_SECONDS'])
 
-    valid, field = is_valid_metadata(decrypted_token)
-    if not valid:
-        raise InvalidTokenException('Missing value {}'.format(field))
+    cleaned_claims = clean_leading_trailing_spaces(decrypted_token)
```

Comment:
I don't think the authenticator module should be taking responsibility for cleaning any of the token data. That seems more appropriate for the metadata_parser module.